_posts/2018-08-29-enable-implicit-provisioning.adoc

```diff
@@ -37,16 +37,16 @@ The device should not automatically provision its credentials. To verify this, l
 1. Load the device credentials on to the device with `aktualizr-cert-provider` command:
 +
 ----
-aktualizr-cert-provider -c credentials.zip -t <device> -d /var/sota
+aktualizr-cert-provider -c credentials.zip -t <device> -d /var/sota/import -r -u
 ----
 +
-You can find the link:https://github.com/advancedtelematic/aktualizr/tree/master/src/cert_provider[`aktualizr-cert-provider` source] in the aktualizr repo.
+You can find the link:https://github.com/advancedtelematic/aktualizr/tree/master/src/cert_provider[`aktualizr-cert-provider` source] in the aktualizr repo. You can also find a compiled binary in the host work directory of bitbake. The path is something like `tmp/work/x86_64-linux/aktualizr-native/1.0+gitAUTOINC+<version>/build/src/cert_provider/aktualizr-cert-provider`.
 
 == Enable implicit provisioning with a Hardware Security Module (HSM)
 
 As described in the link:prod-intro.html[introduction], it's a good idea to use a Hardware Security Model (HSM) to hold potentially sensitive device credentials.
 
-The following procedure describes how to use QEMU to simulate using an HSM. However, the procedure for your HSM will probably be different. We've provided these instructions as a basic guide to how implicit provisioning works but you'll need to make further changes on your own. For example, you'll probably need to adapt your BSP so that aktualizr can access the keys from your HSM.
+The following procedure describes how to use QEMU to simulate using an HSM with the `softhsm` tool. However, the procedure for your HSM will probably be different. We've provided these instructions as a basic guide to how implicit provisioning works but you'll need to make further changes on your own. For example, you'll probably need to adapt your BSP so that aktualizr can access the keys from your HSM.
```

Comment:
We use the "version 2" of this tool and most of the time it's referred as softhsm2. So "softhsm2 tool"?

Reply:
Tricky. The [apparently official website](https://www.opendnssec.org/softhsm/) names it SoftHSM. The recipe in meta-updater is also just "softhsm". Do you think it is likely to cause confusion if we don't specify a version?

Reply:
Ok I won't push more for that. I've just based my "analysis" on `git grep -i softhsm` in aktualizr's repo and most of the matches have a "2" at the end... But the yocto recipe and the official website are probably better authorities on that.

Maybe worth it adding a link to the website at this spot then?